README.md

```diff
@@ -11,4 +11,34 @@ the folder 3rdparty.
 
 Screenshots
 -----------
-![Alan Wakes American Nightmare 1](screenshots/awan1.png)
\ No newline at end of file
+![Alan Wakes American Nightmare 1](screenshots/awan1.png)
+
+
+Building and running on Linux
```

Comment:
What you are proposing here can also be done on OSX and Windows, so I would prefer to simply call it "Building"